mpf/platforms/system11.py

```diff
@@ -44,6 +45,7 @@ def __init__(self, machine: MachineController) -> None:
 
         self.a_side_done_time = 0
         self.c_side_done_time = 0
+        self.debounce_secs = 0
```

Comment:
A number of other values are always pulled by lookup (e.g. `self.system11_config['ac_relay_delay_ms']`), but the debounce time is used every tick during a switchover and I thought it'd be nice to spare all those lookups with a stored value.